src/GoogleMapPersistedInstance.js

```diff
@@ -0,0 +1,58 @@
+import GoogleMap from './GoogleMap';
+
+const clearChildren = (node) => {
+  if (node) {
+    while (node.firstChild) {
+      node.removeChild(node.firstChild);
+    }
+  }
+}
+
+export class GoogleMapPersistedInstance extends GoogleMap {
```

Comment:
Could this functionality become a part of the ```GoogleMap``` component? 
Is there a case where i'd want to use GoogleMap without the persisted instance functionality?